common/lib/xmodule/xmodule/html_module.py

```diff
@@ -420,8 +428,9 @@ class StaticTabFields(object):
     )
 
 
+@edxnotes
```

Comment:
This was added to retain support for edxnotes on static html tabs.  Not sure if these are wanted or not. @symbolist

Reply:
It wasn't there in the original module so we don't need it here either: https://github.com/edx/edx-platform/pull/20954/files#diff-3ba53f005532d5ade732813b11c7911f480221810f6724433f6b5d0c568aa080L400

Reply:
Removed :)